src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -57,21 +58,27 @@ function ExportAnnotations({
   };
 
   return (
-    <>
+    <form
+      className="space-y-3"
+      onSubmit={exportAnnotations}
+      data-testid="export-form"
+    >
       {exportCount > 0 ? (
         <>
-          <p data-testid="export-count">
+          <label data-testid="export-count" htmlFor="export-filename">
```

Comment:
The `label` is required for any input inside a form, otherwise a11y tests won't pass